### ISSUE TITLE
Crashed: ru.yandex.mobile.YandexMobileMetrica.YMMReporter.ivarQueue version 2.9.0

### ISSUE THREAD

**Issue by Ewg777** [yandexmobile/metrica-sdk-ios#63]
version 2.9.0
```
Crashed: ru.yandex.mobile.YandexMobileMetrica.YMMReporter.ivarQueue
0  libobjc.A.dylib                0x219cba66 objc_msgSend + 5
1  MYAPP              0x715ca1 -[YMMFramework functionFromString:] + 171068
2  MYAPP              0x74dbc1 +[YMMWiFiNetworkProvider loadFrameworkIfNeeded] + 400220
3  MYAPP              0x74dc8b +[YMMWiFiNetworkProvider currentNetworks] + 400422
4  MYAPP              0x729aff +[YMMNetworkInfo networkInfoWithReachabilityStatus:] + 252570
5  MYAPP              0x729a7b +[YMMNetworkInfo currentNetworkInfo] + 252438
6  MYAPP              0x712aed -[YMMEventBuilder eventWithClass:ofType:appEnvironment:] + 158344
7  MYAPP              0x7129df -[YMMEventBuilder eventAliveWithAppEnvironment:] + 158074
8  MYAPP              0x72f193 -[YMMReporter endSession:updatedAt:] + 274734
9  MYAPP              0x72f01b -[YMMReporter ensureSessionEnded:] + 274358
10 MYAPP              0x72e7a5 __20-[YMMReporter setup]_block_invoke + 272192
11 libdispatch.dylib              0x21d91823 _dispatch_call_block_and_release + 10
12 libdispatch.dylib              0x21d9e423 _dispatch_queue_drain$VARIANT$mp + 1758
13 libdispatch.dylib              0x21d9da61 _dispatch_queue_invoke$VARIANT$mp + 284
14 libdispatch.dylib              0x21da05e9 _dispatch_root_queue_drain + 1560
15 libdispatch.dylib              0x21d9ffcd _dispatch_worker_thread3 + 96
16 libsystem_pthread.dylib        0x21f55b29 _pthread_wqthread + 1024
17 libsystem_pthread.dylib        0x21f55718 start_wqthread + 8
```

Podfile.lock:
```
  - YandexMobileMetrica (2.9.0):
    - YandexMobileMetrica/Static (= 2.9.0)
  - YandexMobileMetrica/Static (2.9.0)
  - YandexMobileMetricaPush (0.4.0):
    - YandexMobileMetricaPush/Static (= 0.4.0)
  - YandexMobileMetricaPush/Static (0.4.0):
    - YandexMobileMetrica (< 3.0.0, >= 2.6.2)
```

**Comment by fallkrum** [comment]
Have you any information on the steps to reproduce, iOS version? How often do you catch this crash?

**Comment by fallkrum** [comment]
Fixed 2.9.1
